modules/local/cramalign/cram_filter_align_minimap2_filter5end_fixmate_sort/main.nf

```diff
@@ -0,0 +1,57 @@
+process CRAM_FILTER_MINIMAP2_FILTER5END_FIXMATE_SORT {
```

Comment:
I have two concerns:
(1) Does the module now handle multi-library events?
(2) How you would define resource allocation based on genomes with different sizes? please take a look at here: https://github.com/sanger-tol/treeval/blob/microfinder/conf/base.config

Reply:
1) I think it should handle multi-libraries fine - having done some digging, they look to be exactly the same as single library CRAM files, except there are @RG tags defined in the header that are used to annotate each read. So mapping from the CRAM file shouldn't be a problem?
2) That's an oversight on my part - I should define resource usage as in your example there, or as I have it in https://github.com/sanger-tol/metagenomeassembly/blob/3c7bdfb295e5713b9a335192c1321efc7c359ba9/conf/base.config#L77

Reply:
Hi @yumisims, have added resource requirements now!